geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -11,6 +11,7 @@ package org.locationtech.geomesa.hbase.data
 import java.util
 
 import com.google.common.collect.Lists
+import com.google.protobuf.ByteString
```

Comment:
Is this import necessary?  (Maybe the r.size() call pulls it in?)

Reply:
nope, it should be removed, thanks